operations/loyaltyprograms.md

```diff
@@ -82,6 +86,19 @@ Note this operation uses [Pagination](../guidelines/pagination.md).
 | `Id` | string | required | Unique identifier of the loyalty program. |
 | `Name` | string | required | Name of the loyalty program. |
 | `Code` | string | required | Code of the loyalty program. |
+| `Type` | [Loyalty program type](#loyalty-program-type) | string | optional | Type of the loyalty program. |
+| `Subscription` | [Loyalty program subscription](#loyalty-program-subscription) | string | optional | Subscription of the loyalty program. |
```

Comment:
```suggestion
| `Type` | [Loyalty program type](#loyalty-program-type) | string | required | Type of the loyalty program. |
| `Subscription` | [Loyalty program subscription](#loyalty-program-subscription) | string | required | Subscription of the loyalty program. |
```

They are optional right now and we'll not introduce any breaking changes, but we want partners to send us this info, so we should mark it as mandatory in the documentation IMHO 🙏

Reply:
Makes sense 👍🏻  I'll update